packages/api/src/controllers/stream.ts

```diff
@@ -1093,11 +1093,14 @@ app.put(
       }
     }
 
-    const ingest = await getIngestBase(req);
-    await triggerCatalystPullStart(stream, getHLSPlaybackUrl(ingest, stream));
+    if (!stream.isActive) {
+      const ingest = await getIngestBase(req);
+      // TODO run in background
```

Comment:
@thomshutt @victorges not sure if it's worth it but maybe we should run the manifest call in the background in case the stream becomes active whilst we're still waiting for a response there..

Reply:
Good idea. Can we also wait for other states besides Active? I think that as long as we're confident it's in a state that'll become active (e.g Pending) then that's probably okay

Reply:
Oh yeah good plan

Reply:
Ah no, just seen that the 'pending' label you see in studio dashboard comes after the stream is active so that doesn't help.

Reply:
yeah pending is just "active but waiting for stream health data"